### PR TITLE
export visibility of clefs and time signatures

### PR DESCRIPTION
**Pull request by rettinghaus**
This PR adds the `@visible` attribute to clefs that are invisible, and sets `@meter.form` to "invis" if it's hidden.

### CODE REVIEW

**Review by ahankinson — APPROVED** [review]

**Review by th-we — DISMISSED** [review]

File: src/ExportGenerators.mss [hunk 1]
```diff
@@ -1320,6 +1325,10 @@ function GenerateScoreDef (score, barnum) {
     libmei.AddAttribute(scoredef, 'meter.count', timesig.Numerator);
     libmei.AddAttribute(scoredef, 'meter.unit', timesig.Denominator);
     libmei.AddAttribute(scoredef, 'meter.sym', ConvertNamedTimeSignature(timesig.Text));
+    if (timesig.Hidden)
```
[the remark this answers is not in the document]
Comment (rettinghaus) [reply src/ExportGenerators.mss:1328]: This is weird, because with Sibelius 2023.8 it works this way.

[the remark this answers is not in the document]
Comment (th-we) [reply src/ExportGenerators.mss:1328]: I'm a bit wrong here – for the first bar it seems to work. I'll examine this further.

[the remark this answers is not in the document]
Comment (th-we) [reply src/ExportGenerators.mss:1328]: O.K., it *does* work, and I added some things in a pull request to your branch.

**Comment by th-we** [comment]
@rettinghaus Thanks for anschubsing!

**Pull request merged**